docs/success-guide/setup_config_files.md

```diff
@@ -126,25 +126,46 @@ Limits on deletion prevent accidental account deletion in the event of misconfig
 
 
 
-#### Delete Protection*
+#### Delete Protection
 
 If you want to drive account creation and removal through User Sync, and want to manually create a few accounts then you may need this feature to keep User Sync from deleting your manually created accounts.
 
-&#9744; If you need to use this feature, add the lines such as below to the config file at the top level.  Note that this feature is not available at this point in time.
+&#9744; If you need to use this feature, add lines such as below to the config file at the top level.   You can create user groups and add users to that user group on the Adobe Admin Console, and then list that user group as excluded from User Sync processing to protect that set of users.  You can list specific users and/or a pattern that matches specific user names to protect those users.  You can protect users based on their identity type as well.  For example, often User Sync is used only to manage federatedID or enterpriseID user types and you can exclude adobeID type users from management by User Sync.  You only need to include configuration items for exclusions that you wish to use.
```

Comment:
this tripped me twice now, so let's fix it. :)  You say "You can create user groups and add users to that user group on the..."  Aside from the plural singular mismatch, the sentence is just too long.  I suggest "To protect users on the Admin Console from updates, create a user group and put the protected users into that group, then list that group as excluded from User Sync processing."